dallinger/mturk.py

```diff
@@ -182,104 +248,112 @@ def get_qualification_type_by_name(self, name):
 
     def assign_qualification(self, qualification_id, worker_id, score, notify=False):
         """Score a worker for a specific qualification"""
-        return self._is_ok(self.mturk.assign_qualification(
-            qualification_id,
-            worker_id,
-            score,
-            notify
+        return self._is_ok(self.mturk.associate_qualification_with_worker(
+            QualificationTypeId=qualification_id,
+            WorkerId=worker_id,
+            IntegerValue=score,
+            SendNotification=notify
         ))
 
-    def get_current_qualification_score(self, name, worker_id):
-        """Return the current score for a worker, on a qualification with the
-        provided name.
+    def set_qualification_score(self, qualification_id, worker_id, score, notify=False):
+        """BBB only.
         """
-        qtype = self.get_qualification_type_by_name(name)
-        if qtype is None:
-            raise QualificationNotFoundException(
-                'No Qualification exists with name "{}"'.format(name)
-            )
+        return self.assign_qualification(qualification_id, worker_id, score, notify)
 
-        for_type = self.mturk.get_all_qualifications_for_qual_type(qtype['id'])
-        for_worker = [q for q in for_type if q.SubjectId == worker_id]
-        if for_worker:
-            return {
-                'qtype': qtype,
-                'score': int(for_worker[0].IntegerValue)
-            }
-        return {
-            'qtype': qtype,
-            'score': None
-        }
+    def update_qualification_score(self, qualification_id, worker_id, score):
+        """BBB only"""
+        return self.assign_qualification(qualification_id, worker_id, score)
```

Comment:
Should we log a deprection warning? Or just do e.g. `update_qualification_score = assign_qualification` to avoid the need for coverage?